src/ConcealWallet/ConcealWallet.cpp

```diff
@@ -1018,12 +701,9 @@ bool simple_wallet::new_wallet(const std::string &wallet_file, const std::string
       return false;
     }
 
-    try {
-      cn::WalletHelper::storeWallet(*m_wallet, m_wallet_file);
-    } catch (std::exception& e) {
-      fail_msg_writer() << "failed to save new wallet: " << e.what();
-      throw;
-    }
+    logger(logging::INFO) << "Saving wallet...";
+    m_chelper.save_wallet(*m_wallet, m_wallet_file, logger);
+    logger(logging::INFO, logging::BRIGHT_GREEN) << "Saved successful";
```

Comment:
Why not call logger directly inside the function to avoid duplicating the 2 messages each time?
```suggestion
    m_chelper.save_wallet(*m_wallet, m_wallet_file, logger);
```